go.mod

```diff
@@ -32,4 +32,5 @@ require (
 	k8s.io/client-go v0.17.2
 	k8s.io/utils v0.0.0-20200414100711-2df71ebbae66 // indirect
 	sigs.k8s.io/controller-runtime v0.4.0
+	sigs.k8s.io/controller-tools v0.2.9 // indirect
```

Comment:
Mmm.. not sure where this comes from? we need controller-gen 0.2.9, not controller-tools.
Also it's installed locally and not as a dependency.
Does it stay if you run `go mod tidy`?

Reply:
It goes away after running `go mod tidy`

Reply:
commit and push it?

Reply:
yep